### PR TITLE
Refactoring/final clean up

### PR DESCRIPTION
**Pull request by e-spitz**
Just realized we could be doing PR templates here =) Next project!

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -32,67 +31,38 @@ var savedCovers = [
 var currentCover;
 
 //EVENT LISTENERS
-window.addEventListener("load", displayRandomHomePage);
-randomCoverButton.addEventListener("click", displayRandomHomePage);
+window.addEventListener("load", displayRandomBook);
```
Comment (janitastic) [inline src/main.js:34]: This makes so much more sense! Glad we did it!

File: src/main.js [hunk 2]
```diff
@@ -32,67 +31,38 @@ var savedCovers = [
...
-
-function getRandomImage() {
-  var image = covers[Math.floor(Math.random() * covers.length)]
-  return image;
-}
-function displayRandomImage() {
-  htmlImage.src = getRandomImage();
-  // console.dir(htmlImage);
-  // console.log(htmlImage);
+// FUNCTIONS
```
Comment (janitastic) [inline src/main.js:42]: Woohoo! We got the original function to work!

File: src/main.js [hunk 3]
```diff
@@ -32,67 +31,38 @@ var savedCovers = [
...
+  return Math.floor(Math.random() * array.length);
 }
 
-function getRandomTitle() {
-  var title = titles[Math.floor(Math.random() * titles.length)]
-  return title;
-}
-function displayRandomTitle() {
-  htmlTitle.innerText = getRandomTitle();
+function displayRandomBook() {
```
Comment (janitastic) [inline src/main.js:47]: Great understanding of how to use the data file here!

File: src/main.js [hunk 4]
```diff
@@ -32,67 +31,38 @@ var savedCovers = [
...
-function displayRandomTagline1() {
-  htmlTagline1.innerText = getRandomTagline1();
+function displayCurrentCover() {
+  htmlImage.src = currentCover.cover;
+  htmlTitle.innerText = currentCover.title;
+  htmlTagline1.innerText = currentCover.tagline1;
+  htmlTagline2.innerText = currentCover.tagline2;
 }
 
-function getRandomTagline2() {
```
Comment (janitastic) [inline src/main.js:71]: Yay! We cleaned up so much code!

**Review by janitastic — COMMENTED** [review]
We're getting so good a refactoring! Thanks for looping in Nathan!

File: src/main.js (hunk 4 above)
Comment (e-spitz) [reply src/main.js:71]: SO fresh, SO clean.

**Pull request merged**